package.json

```diff
@@ -3,7 +3,7 @@
   "version": "3.5.6",
   "description": "A minimal node SOAP client",
   "engines": {
-    "node": ">=8.11.1"
+    "node": "16 || 18 || 20 || 21"
```

Comment:
This more explicit versioning aligns with the strategy with do with LoopBack 4.

Note that for non-LTS versions (i.e. odd-numbered versions), we will drop their support and it won't be considered a breaking change.